src/App.jsx

```diff
@@ -73,15 +79,18 @@ const App = () => {
      * @function fetchData
      */
     async function fetchData() {
-      await generateUsersList(session);
-      await generateActivityTTL(session);
-      await updateUserActivity(session);
-      await createDocumentContainer(session);
+      let podUrl = (await getPodUrlAll(session.info.webId, { fetch: session.fetch }))[0];
+      podUrl = podUrl || session.info.webId.split('profile')[0];
+      setSignedInPod(podUrl);
+      await updateUserActivity(session, podUrl);
+      await createDocumentContainer(session, podUrl);
       await createOutbox(session);
       try {
-        let listUsers = await getUsersFromPod(session);
-        setUserList(listUsers);
+        setLoadingUsers(true);
+        setLoadingActive(true);
```

Comment:
These states were already set to true when initialized.

Reply:
updated